frontend/src/v5/store/containers/containers.sagas.ts

```diff
@@ -65,49 +70,50 @@ export function* removeFavourites({ containerId, teamspace, projectId }: RemoveF
+		const stats = yield statsQueue.enqueue(teamspace, projectId, containerId);
+		
+		const basicDataEqual = compByColum(['unit', 'type'])(container, stats);
+		// eslint-disable-next-line max-len
+		const revisionsEqual = (container?.latestRevision === stats?.revisions?.latestRevision && container?.status === stats?.status) // if it has revisions
+							|| (container?.latestRevision === '' && stats?.revisions?.total === 0); // if it doesnt
 
 		// Only update if theres is new data
-		if (isPending || !isEqualWith(storedContainers, containersWithoutStats, compByColum(['_id', 'name', 'role', 'isFavourite']))) {
-			yield put(ContainersActions.fetchContainersSuccess(projectId, containersWithoutStats));
+		if (!basicDataEqual || !revisionsEqual) {
+			yield put(ContainersActions.fetchContainerStatsSuccess(projectId, containerId, stats));
 		}
-
-		yield all(
-			containers.map(
-				(container) => put(ContainersActions.fetchContainerStats(teamspace, projectId, container._id)),
-			),
-		);
 	} catch (error) {
 		yield put(DialogsActions.open('alert', {
-			currentActions: formatMessage({ id: 'containers.fetchAll.error', defaultMessage: 'trying to fetch containers' }),
+			currentActions: formatMessage({ id: 'containers.fetchStats.error', defaultMessage: 'trying to fetch containers details' }),
 			error,
 		}));
 	}
 }
 
-export function* fetchContainerStats({ teamspace, projectId, containerId }: FetchContainerStatsAction) {
+export function* fetchContainers({ teamspace, projectId }: FetchContainersAction) {
 	try {
-		const stats = yield API.Containers.fetchContainerStats(teamspace, projectId, containerId);
+		const { containers }: FetchContainersResponse = yield API.Containers.fetchContainers(teamspace, projectId);
+		const containersWithoutStats = prepareContainersData(containers);
+		const storedContainers = yield select(selectContainers);
+		const isPending = yield select(selectIsListPending);
 
-		const container = yield select(selectContainerById, containerId);
+		// Only update if theres is new datad
+		if (isPending || !isEqualWith(storedContainers, containersWithoutStats, compByColum(['_id', 'name', 'role', 'isFavourite']))) {
+			yield put(ContainersActions.fetchContainersSuccess(projectId, containersWithoutStats));
+		}
 
-		const basicDataEqual = compByColum(['unit', 'type'])(container, stats);
-		// eslint-disable-next-line max-len
-		const revisionsEqual = (container?.latestRevision === stats?.revisions?.latestRevision && container?.status === stats?.status) // if it has revisions
-							|| (container?.latestRevision === '' && stats?.revisions?.total === 0); // if it doesnt
+		statsQueue.resetQueue();
 
-		// Only update if theres is new data
-		if (!basicDataEqual || !revisionsEqual) {
-			yield put(ContainersActions.fetchContainerStatsSuccess(projectId, containerId, stats));
-		}
+		yield all(
+			containers.sort(getSortingFunction({ column: ['name'], direction:[SortingDirection.DESCENDING] })).map(
```

Comment:
Because we want to fetch the data in the same order the user sees it
If it is not ordered in the ordered list in the containers list it can look like this


| loading container 0 |
| container 1 loaded |
| loading container 2 |
| loading container 3 |
| loading container 4 |
| container 5 loaded |
| container 6 loaded |

this way it looks like this
| container 0 loaded |
| container 1 loaded |
| container 2 loaded |
| loading container3 |
| loading container 4 |
| loading container 5 |
| loading container 6 |

Reply:
This will only be the case if the alphabetically sorted list matches the lastUpdated sorted list because the containers list is sorted by lastUpdated by default. Is it possible to do the same sort here?

You can see this when loading the containers in a project other than the generated 1000 containers one

Reply:
Is not possible because the date comes with the stats